checker/tests/run-pass/test_source.rs

```diff
@@ -31,10 +31,9 @@ fn no_summary_analyzed_anyway() {
         }
     }
     let d: &dyn Dynamic = &S {} as &dyn Dynamic; // forget type info of S
-    verify!(d.f(1) == 3); // normally, this statement would disable verification for this
-                          // function and we would not see the message below. With --diag=strict,
-                          // we do not see an error here (as d.f is uninterpreted), but we still
-                          // see the below error.
+
+    // With --diag=strict we still see an error even though d.f is uninterpreted.
```

Comment:
That kind of reverses the purpose of this test. It's purpose was to test that there is no diag for the uninterpreted function call, but still for others in this function.